jwst/dq_init/dq_init_step.py

```diff
@@ -1,8 +1,8 @@
 #! /usr/bin/env python
 
-from jwst.stpipe import Step
-from jwst import datamodels
-from jwst.dq_init import dq_initialization
+from ..stpipe import Step
+from .. import datamodels
+from .. import dq_initialization
```

Comment:
I think this one still needs to be updated to "from . import dq_initialization", because it's part of the dq_init step.